packages/devtools_app/lib/src/trees.dart

```diff
@@ -106,6 +106,9 @@ class TreeNode<T extends TreeNode<T>> {
     _isExpanded = false;
   }
 
+  /// Override to handle pressing on a Leaf node.
```

Comment:
is this `onSelectLeaf`, `selectLeaf`, or similar rather than `leaf`?

Reply:
Fyi @kenzieschmoll can you take a look at the tree changes to support leaf nodes as well?

Reply:
Mimicked to be like expand and collapse.  I would have preferred onSelectLeaf but thought it made more sense to be consistent with rest of tree.